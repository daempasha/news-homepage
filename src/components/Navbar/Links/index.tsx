import links from "@components/links";

const Links = () => {
    return <div className="hidden md:block">
        <ul data-testid="links">
            {links.map((category) =>
                <li className="inline mx-5 text-gray-800"><a className="hover:text-orange-600 " href="#">{category}</a></li>
            )}

        </ul>
    </div>
}

export default Links;