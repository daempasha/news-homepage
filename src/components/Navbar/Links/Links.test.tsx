import { describe, it, expect } from 'vitest';
import { render } from '@testing-library/react';
import Links from './index';

describe('<Links />', () => {
    it('renders correctly', () => {
        const links = render(<Links />)
        expect(links).toMatchSnapshot()
    });

});