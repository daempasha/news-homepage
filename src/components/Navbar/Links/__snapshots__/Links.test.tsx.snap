// Vitest Snapshot v1

exports[`<Links /> > renders correctly 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="hidden md:block"
      >
        <ul
          data-testid="links"
        >
          <li
            class="inline mx-5 text-gray-800"
          >
            <a
              class="hover:text-orange-600 "
              href="#"
            >
              Home
            </a>
          </li>
          <li
            class="inline mx-5 text-gray-800"
          >
            <a
              class="hover:text-orange-600 "
              href="#"
            >
              New
            </a>
          </li>
          <li
            class="inline mx-5 text-gray-800"
          >
            <a
              class="hover:text-orange-600 "
              href="#"
            >
              Popular
            </a>
          </li>
          <li
            class="inline mx-5 text-gray-800"
          >
            <a
              class="hover:text-orange-600 "
              href="#"
            >
              Trending
            </a>
          </li>
          <li
            class="inline mx-5 text-gray-800"
          >
            <a
              class="hover:text-orange-600 "
              href="#"
            >
              Categories
            </a>
          </li>
        </ul>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="hidden md:block"
    >
      <ul
        data-testid="links"
      >
        <li
          class="inline mx-5 text-gray-800"
        >
          <a
            class="hover:text-orange-600 "
            href="#"
          >
            Home
          </a>
        </li>
        <li
          class="inline mx-5 text-gray-800"
        >
          <a
            class="hover:text-orange-600 "
            href="#"
          >
            New
          </a>
        </li>
        <li
          class="inline mx-5 text-gray-800"
        >
          <a
            class="hover:text-orange-600 "
            href="#"
          >
            Popular
          </a>
        </li>
        <li
          class="inline mx-5 text-gray-800"
        >
          <a
            class="hover:text-orange-600 "
            href="#"
          >
            Trending
          </a>
        </li>
        <li
          class="inline mx-5 text-gray-800"
        >
          <a
            class="hover:text-orange-600 "
            href="#"
          >
            Categories
          </a>
        </li>
      </ul>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
